GNUmakefile

```
@@ -11,9 +12,9 @@ prep: phony
 build: phony
 	@go build -o ${BINARY}
 
-dev: phony build
-	@mkdir -p ~/.packer.d/plugins/
-	@mv ${BINARY} ~/.packer.d/plugins/${BINARY}
+dev:
```

```suggestion
dev: phony
```

Needs a dependency on `phony`. See lines 54-60 for details